styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  font-family: "Arial", sans-serif, sans-serif;
+  background-color: #f4f4f4;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+#quizForm {
```

Comment:
You can add `max-width` to this element. Forms should be rather compact, not stretching through the entire page. When it is more compact it's easier to fill and read.